src/lib/types/EventTypes.ts

```diff
@@ -17,4 +17,8 @@ export enum EventTypes {
    * A new message is sent to a specific conversation
    */
   ConversationMessage = 'conversationMessage',
+  /**
+   * A inboxId or conversation has been approved or denied
```

Comment:
is that a correct comment?

Reply:
Yes you get a consent stream whenever an inboxId or conversation has been approved or denied.